saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/suggester.js

```diff
@@ -12,29 +12,69 @@ var Suggester = (function() {
     var space = {},
     	selItem = [],
     	getUrl = [],
+    	getValue = [],
     	
     	maxElementsInList = 10,
-    	cssClass = "suggestions";
+    	cssClass = "suggestions",
     
+    	SELECTED_FIELD_CLASS = "suggestion",
+    	CONTAINER_ID = "suggestions",
+    	AREA_ID = "suggestion-area",
+    	WRAPPER_ID = "suggestion-wrapper",
+    	
+		FIELD_FOR = "data-field-for",
+    		
+    		
+    	OFFSET = 100;
     
     /* Initialize url parameter and runs main methods: create suggestion container
      * and assign load suggestions methods to input field.
      * 
      * @param $field input field connected to suggester
      * @param source configuration parameters(json):
-     * 		- getUrl - function that takes one @param id and returns url location with list of suggestions.
-     * 		  Module requires function instead of simple url, because this way it can handle url like e.g. "/keywords/COMMON/{phrase}/list"
+     * 		- boxyMode
+     * 			- enabled - turn on boxy mode,
+     * 			- placeholder - text that is used as placeholder in textarea  
+     * 
+     * 		- url - returns url location with list of suggestions.
+     * 		   		it can be function or simple string url, function can handle url like e.g. "/keywords/COMMON/{phrase}/list"
+     *		- getValue - (function) returns value of suggestion list elements e.g.
```

Comment:
i understand it is the function that will be invoked on every element of the suggestion list to get its value?